package.json

```diff
@@ -1,6 +1,13 @@
 {
```

Comment:
While it looks like we added a bunch of new dependencies, most of these are _not_ new. With yarn v1, we could directly depend on artifacts from transitive dependencies. pnpm does not let you do this, so we have to add add those previously-transitive dependencies that we currently directly depend on.